Enterprise-CMCS/eAPD
# Nz/4502 [Bug] Create APD button not being enabled when switching from MMIS to HITECH

Resolves #4502 

### Description
If you try to create a MMIS APD then switch to creating a HITECH APD, even if all fields are filled out the Create APD button won't be enabled. This doesn't happen when switching from HITECH to MMIS.

### Significant changes or possible side effects

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| New APD Page | Creating an MMIS APD but then switching to HITECH before submitting | the button enables | jest |
| ennableMmis | is false | user can create a HITECH APD | jest |

### Steps to manually verify this change

1. Try to create a MMIS APD
2. Fill out fields and verify Create APD button is enabled
3. Switch APD type to HITECH
4. Verify the Create APD button is disabled regardless if all fields are filled out

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
